tools/start.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+
+import {chalkError, chalkSuccess} from "./chalkConfig";
+import fs from 'fs';
+import dotenv from 'dotenv';
```

Comment:
where is `dotenv` coming from? It doesn't appear to be in the package.json

Reply:
Or is just installed globally like fs? This script runs locally so nevermind

Reply:
I think it's a dependency of `dotenv-webpack`.  Once you do an `npm install` it should show in `node_modules` folder. Also, should be located in the `package-lock.json`.